tests/src/JIT/Regression/JitBlue/GitHub_19022/GitHub_19022.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Numerics;
+
+namespace JitCrashPOC
+{
+    class Program
+    {
+        static public int s_res;
+
+        static int Main(string[] args)
+        {
+            var map = new ItemRunner();
+
+            s_res = 0;
+            map.UpdateItem(0,10);            
+
+            if (s_res == 300)
+            {
+                Console.WriteLine("Passed");
+                return 100;
+            }
+            else
+            {
+                Console.WriteLine("Failed");
+                return 101;
+            }
+        }
+    }
+
+    class Item
+    {
+        public Vector3 _Position = new Vector3(0.0f, 0.0f, 0.0f);
+    }
+
+    class ItemRunner
+    {
+        public ItemRunner()
+        {
+            for (int i = 0; i < _Pool.Length; ++i) { _Pool[i] = new Item(); }
+        }
+
+        private const float _LenghtZ = 1000.0f;
+
+        private static readonly Vector3 _Start = new Vector3(0.0f, -1021.7f, -3451.3f);
```

Comment:
Should we also have tests for `Vector128<T>` and `Vector256<T>`, which are similar to, but sometimes handled separately from `Vector2`/`Vector3`/`Vector4`?

Reply:
I don't think that adding more tests would add any additional code-execution coverage here.

Reply:
OSX test failure:
`13:35:25 FAILED   - JIT/HardwareIntrinsics/X86/Avx/Avx_ro/Avx_ro.sh`
Is this a known issue?

Reply:
Yes. That is: https://github.com/dotnet/coreclr/issues/19001, which @fiigii was looking at.